### ISSUE TITLE
Missing "simple_history.templatetags" value in setup.py packages list (#197, branch django19)

### ISSUE THREAD

**Issue by sgallen** [treyhunner/django-simple-history#200]
In testing the django19 branch, I've noticed that If you pip install it e.g.:

`pip install git+https://github.com/treyhunner/django-simple-history.git@django19`

Then enable a model, and click the `History` button from within the admin, you get the following error:

```
TemplateSyntaxError at /admin/swarm/campaignstatus/1/history/
'simple_history_compat' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_static
admin_urls
cache
future
i18n
l10n
log
static
staticfiles
tz
```

The core of the issue seems to be that the installed package is missing the `simple_history/templatetags/` directory.  This can be remedied by modifying the `setup.py` file at the root of the repo as follows:

`packages=["simple_history", "simple_history.templatetags", "simple_history.management", "simple_history.management.commands"],`

**Comment by macro1** [comment]
Scott: Thanks for taking the time to try out that branch. I had completely
forgotten we don't make use of the `find_packages()` tool in
simple-history. I'll be sure to get that in before it's merged.

**Comment by macro1** [comment]
Resolved in #197
